.circleci/config.yml

```diff
@@ -68,13 +68,18 @@ jobs:
             fi
           environment:
             JEST_JUNIT_OUTPUT_DIR: ./coverage/junit/
+      - run:
+          name: Install playwright
+          command: npx playwright install
```

Comment:
Why are we using npx to install this? Can we have it through npm dependencies instead?

Reply:
the name is misleading which is from their docs. 
it is actually installing supported browsers.
I will fix it to install supported browsers.
https://playwright.dev/docs/ci